modules/ROOT/pages/queries/select-version.adoc

```diff
@@ -9,16 +9,16 @@ The Cypher version can be set for individual queries or as the default language
 == Cypher 5, Cypher 25, and Neo4j explained
 
 Starting in 2025, the Neo4j server transitioned to a calendar-based versioning system.
-This means Neo4j will no longer use its previous semantic versioning and release pattern (e.g., 5.25, 5.26).
+This means Neo4j will no longer use its previous semantic versioning and release pattern (e.g., 5_25, 5.26).
```

Comment:
why?

Reply:
both as in why change to underscore at all and if that is indeed wanted, why only for one of them?

Reply:
I honestly have no idea how that got there. Will change